tests/__init__.py

```diff
@@ -13,7 +13,7 @@
     "host",
     "image",
     "ingredients",
-    "instructions",
+    # "instructions",
```

Comment:
Similar to `ingredients` and `ingredient_groups`, it seems redundant that we require _both_ `instructions` and `instructions_list` to have test coverage.

The latter is easier to write JSON test cases for, in my opinion.

Reply:
If we make this change, I think we should also add some test coverage to check that `instructions` and `instructions_list` contain equivalent content.

Inter-instruction newlines/whitespace should generally be the only difference.